pomm-project/pomm-bundle/2.0/etc/pomm.yaml

```diff
@@ -0,0 +1,10 @@
+pomm:
+    configuration:
+        my_db1:
+            dsn: '%env(DATABASE_DSN)%'
+#            pomm:default: true
```

Comment:
In other PRs, we were discussing about the utility of adding commented config. @stof said that it's better to not add this commented config. Others think it's OK. I'm divided. Maybe @fabpot can share his opinion about this.